rerun_cpp/src/rerun/recording_stream.cpp

```diff
@@ -91,6 +94,41 @@ namespace rerun {
         rr_recording_stream_flush_blocking(_id);
     }
 
+    Error RecordingStream::try_log_components(
```

Comment:
`components` vs. `component_batches` mean very different things :grimacing: 
```suggestion
    Error RecordingStream::try_log_component_batches(